### DISCUSSION TITLE
[4.2.8] changeData后scale无法自动适配

### DISCUSSION THREAD

**Discussion by YuzKio** [antvis/G2#5770] [Q&A]
更新前数据的最大值为1000，更新后数据的最大值为2000。使用changeData更新后tick的最大值仍是1000，而柱状图会超出图。如何进行适配？

**Comment by hustcc** [comment]
应该不太会，除非你写死了 scale 的最大值。来求个代码调试一下。

**Reply by YuzKio** [reply]
https://codesandbox.io/p/sandbox/crimson-thunder-v2y669?file=%2Findex.ts%3A56%2C32 主要是和scrollbar配合使用的时候会出现这个情况。（scrollbar似乎有不少坑
